content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -375,6 +377,18 @@ Changes the [PaymentChannelCreate transaction][] type so that it adds new [payme
 This change makes it easier to look up payment channels by their recipient and prevents accounts from being deleted if they are the recipient for open payment channels (except those channels created before this amendment).
```

Comment:
This sentence has multiple clauses and can be hard to parse to the end. I suggest breaking it up a bit.

Reply:
That's a good point, but I'm going to leave this change for later since it's not related to the v1.5.0 release.

Reply:
On closer examination, this _is_ related to v1.5.0 because v1.5.0 has an API change to make the `account_channels` method backwards-compatible with how it worked before this amendment... which means this amendment no longer makes it easier to look up payment channels by their recipient, from an API user perspective. This amendment still restructures the ledger a little to make those lookups easier, but the plan is to introduce dedicated API functionality to do that in a later version, instead of providing that functionality in an unintentionally-not-backwards-compatible way.